templates/admin/src/components/settings.jsx.ts

```diff
@@ -0,0 +1,149 @@
+import { TemplateFunction } from "../../../../src/lib/createAdapter";
```

Comment:
I would suggest to use one template for JSX and TSX, there are only 8 different sections and most of the code is the same.

Reply:
... similar to index.tsx_jsx.ts